xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/extension.vm

```diff
@@ -1038,6 +1069,24 @@ $namespace##
     </div>
     <div class="extension-header">
       <h2 class="extension-title">
+        #if($!extensionStatus == 'core' ||
```

Comment:
I don't like too much this duplicated logic. @tmortagne do you know if we have velocity services for this?

Reply:
AFAIK, this concept of status is something @mflorea introduced in `#determineExtensionStatus`, and it does not exist on Java side.

Reply:
Note that the logic is not duplicated, it looks very similar, but e.g. there's also the `incompatible` ones for dependencies.
This solution is very similar to what was done in CSS before (see the css removed from extension.css, it wasn't really refined either ^^' ). Instead of having a switch in the rulesets, I pretty much brought it in the template. I'd need to get back to it, but a more elegant solution would be to store the icon mapping in a unified table. This was just a minimal working alternative with icons actually in the HTML template and not only in the CSS.